### DISCUSSION TITLE
gemoji is missing emojis that work in GitHub User Interface

### DISCUSSION THREAD

**Discussion by zrosenbauer** [remarkjs/remark#1238] [Q&A] [answered]
So I found that some emojis aren't translated from shortcodes to emojis...

```bash
✅  :smile: => 🙂
🟥  :bowtie: => :bowtie: 
```

As I dug I found that this is not an issue with remark-gemoji (or even [wooorm/gemoji](https://github.com/wooorm/gemoji)) it seems to be with the GitHub lib itself => [github/gemoji](https://github.com/github/gemoji).

I did find that `:bowtie:` is present in the 2.0 but the output/content is empty string => https://github.com/github/gemoji/blob/2.0-stable/db/emoji.json.

I've attached an output HTML file that I translated using remark-rehype:
[emojis.html.txt](https://github.com/remarkjs/remark/files/12834409/emojis.html.txt)

Any ideas on how to solve this? I'm happy to open a PR or do the leg work but I haven't dug deep and I hope that @wooorm  or someone else in community can point me the right direction.

**Comment by wooorm** [comment, marked as answer]
> Any ideas on how to solve this?

Solve what?

There were several GH-specific gemoji, that were unrelated to unicode, and so only worked on their platform.
They removed most, but some common ones remain.

**Reply by zrosenbauer** [reply]
Got yea I didn't realize that they weren't unicode emojis (I just assumed they would be...).

So to support those, I just add a handler and replace them with images 👍
